Library/Homebrew/os/mac/keg.rb

```diff
@@ -50,6 +50,17 @@ def change_rpath(old, new, file)
     raise
   end
 
+  def delete_rpath(rpath, file)
```

Comment:
Other methods here do
```ruby
@require_relocation = true
```
upon invocation (after checking there's something to do), but I don't think that's necessary here.